core/src/main/java/hudson/PluginManager.java

```diff
@@ -437,8 +626,9 @@ private boolean containsHpiJpi(Collection<String> bundledPlugins, String name) {
     /**
      * Returns the manifest of a bundled but not-extracted plugin.
      */
+    @Deprecated // See https://groups.google.com/d/msg/jenkinsci-dev/kRobm-cxFw8/6V66uhibAwAJ
```

Comment:
change the javadoc and point to a replacement - or say what the code using this should use.